src/components/Home.js

```diff
@@ -45,13 +45,14 @@ class Home extends Component {
     const { navigator } = this.props;
 
     const { height, width } = Dimensions.get('window');
-    const headerHeight = height * 0.75;
+    const headerHeight = height * 0.88;
 
     const goToFilter = () => navigator.push({name: 'Filter', sceneConfig: Navigator.SceneConfigs.VerticalUpSwipeJump});
 
     var filterText = 'do something';
     if (this.props.viewer.activeFilterCategories.edges.length > 0) {
-      filterText = this.props.viewer.activeFilterCategories.edges.map(n => n.node.name).join(' & ');
+      //Remove here the two others, just keep 1 and the +
+      filterText = this.props.viewer.activeFilterCategories.edges.map(n => n.node.name).join('+ ');
```

Comment:
This won't work. If you like to only have the first one and then a plus you should do 
`filterText = this.props.viewer.activeFilterCategories.edges[0].node.name + '+';`

But I'm not really sure if this would be the way I like to have it but let's do it for the beta.